docs/source/delta-managed-commit.md

```diff
@@ -0,0 +1,92 @@
+
+
+# DynamoDB Commit Owner
+
+<Delta> 4.0-preview also introduces a DynamoDB backed Commit Owner implementation.
+
+## Quickstart Guide
+
+### 1. Create the DynamoDB table
+The DynamoDB Commit Owner requires a backend DynamoDB table to manage commits. One DynamoDB table can be used to manage multiple Delta tables. You have the choice of creating the DynamoDB table yourself (recommended) or having it created for you automatically.
+
+- Creating the DynamoDB table yourself
+
+    This DynamoDB table will maintain commit metadata for multiple Delta tables, and it is important that it is configured with the [Read/Write Capacity Mode](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html) (for example, on-demand or provisioned) that is right for your use cases. As such, we strongly recommend that you create your DynamoDB table yourself. The following example uses the AWS CLI. To learn more, see the [create-table](https://docs.aws.amazon.com/cli/latest/reference/dynamodb/create-table.html) command reference.
+
+    ```bash
+    aws dynamodb create-table \
+    --region us-west-2 \
+    --table-name dynamodb_delta_commit_owner \
+    --attribute-definitions AttributeName=tableId,AttributeType=S
+    --key-schema AttributeName=tableId,KeyType=HASH
+    --billing-mode PAY_PER_REQUEST
+    ```
+
+- Automatic DynamoDB table creation
+
+    When you create a DynamoDB Managed Table, the commit owner will try to create a backing DynamoDB table if it does not exist. This default table supports 5 strongly consistent reads and 5 writes per second. You may change these default values using the table-creation-only configurations keys `spark.databricks.delta.managedCommits.commitOwner.ddb.writeCapacityUnits` and `spark.databricks.delta.managedCommits.commitOwner.ddb.readCapacityUnits`.
+
+### 2. Launch a spark shell with the right packages:
+
+```bash
+bin/spark-shell \
+--packages io.delta:delta-spark_2.13:4.0.0,org.apache.hadoop:hadoop-aws:3.3.4 \
+--repositories https://oss.sonatype.org/content/repositories/iodelta-1147 \
+--conf spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension \
+--conf spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog \
+--conf spark.databricks.delta.managedCommits.commitOwner.ddb.awsCredentialsProviderName=<credentialsProviderName>
+```
+
+`<credentialsProviderName>` must be the fully qualified class name of the [AWS Credentials Provider](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AWSCredentialsProvider.html) (e.g. `com.amazonaws.auth.profile.ProfileCredentialsProvider`) which must be used for authenticating with the desired DynamoDB instance.
+
+### 3. Create a table with DynamoDB as the commit owner by running the following command:
```

Comment:
This is a title o a subsection.

```suggestion
### 3. Creating a managed-commit table with DynamoDB as the commit owner
```

Reply:
In that case:
"### 1. Create the DynamoDB table" --> "### 1. Creating the DynamoDB table"

"### 2. Launch a spark shell with the right packages:" --> "### 2. Launching the spark shell"
?